README.md

```diff
@@ -10,9 +10,9 @@ A golang library for the [COSE specification][cose-spec]
 
 **Current Release**: [go-cose alpha 1][release-alpha-1] 
 
-The project was *initially* forked from the  upstream [mozilla/go-cose][mozilla-go-cose] project, however it was agreed between the veraison and mozilla maintainers to retire mozilla/go-cose project and focus on [veraison/go-cose][veraison-go-cose] as the active project.
+The project was *initially* forked from the  upstream [mozilla-services/go-cose][mozilla-go-cose] project, however it was agreed between the Veraison and Mozilla maintainers to retire mozilla-services/go-cose project and focus on [veraison/go-cose][veraison-go-cose] as the active project.
```

Comment:
I prefer using the active phrasing (i.e., as it was before) to the passive construction.

Reply:
Nothing wrong in introducing with a brief background of the initial state (earlier) and then thanking the original contributors (in present) and then moving on. In my view it flows well.

Reply:
what I am referring to is the change from the original:
```
the veraison and mozilla maintainers have agreed to [...]
```
to
```
it was agreed between the Veraison and Mozilla maintainers to [...]
```
(the former is better.)

Reply:
But that was correct when the action was in progress, like I have agreed to hand over XYZ to you.

But once the handover happened that should also go to past tense, like It was agreed between UK and EU to part ways back in 2019.

Reply:
But it's not that the agreement has vanished.  Or so I hope :-)